classes/activity-manager.js

```diff
@@ -146,8 +147,10 @@ class ActivityManager {
      * @private
      */
     static parseRole(member, role, activityName) {
-        let stampNumber = parseInt(role.name.split('#')[1]);
-        let newRoleID = discordServices.stampRoles.get(stampNumber + 1);
+        let stampNumber = discordServices.stampRoles.get(role.id);
+        console.log(stampNumber);
+        let newRoleID = discordServices.stampRoles.findKey(number => number === (stampNumber + 1));
+        console.log(newRoleID);
 
         if (newRoleID != undefined) {
```

Comment:
Lets add an else with a message to user saying they have the highest stamp available! @mwang2000

Reply:
done and deleted those console.logs i accidentally left in lol